src/TestUtils/src/UITest.Appium/HelperExtensions.cs

```diff
@@ -145,6 +192,84 @@ public static void DoubleClick(this IApp app, string element)
 			});
 		}
 
+		/// <summary>
+		/// Performs a mouse double click on the given coordinates.
+		/// </summary>
+		/// <param name="app">Represents the main gateway to interact with an app.</param>
+		/// <param name="x">The x coordinate to double click.</param>
+		/// <param name="y">The y coordinate to double click.</param>
+		public static void DoubleClickCoordinates(this IApp app, float x, float y)
+		{
+			app.CommandExecutor.Execute("doubleClickCoordinates", new Dictionary<string, object>
+			{
+				{ "x", x },
+				{ "y", y }
+			});
+		}
+
+		/// <summary>
+		/// Performs two quick tap / touch gestures on the matched element.
+		/// </summary>
+		/// <param name="app">Represents the main gateway to interact with an app.</param>
+		/// <param name="element">Target Element.</param>
+		public static void DoubleTap(this IApp app, string element)
+		{
+			var elementToClick = app.FindElement(element);
+			app.CommandExecutor.Execute("doubleTap", new Dictionary<string, object>
+			{
+				{ "element", elementToClick },
+			});
+		}
+
+		/// <summary>
+		/// Performs two quick tap / touch gestures on the given coordinates.
+		/// </summary>
+		/// <param name="app">Represents the main gateway to interact with an app.</param>
+		/// <param name="x">The x coordinate to double tap.</param>
+		/// <param name="y">The y coordinate to double tap.</param>
+		public static void DoubleTapCoordinates(this IApp app, float x, float y)
+		{
+			app.CommandExecutor.Execute("doubleTapCoordinates", new Dictionary<string, object>
+			{
+				{ "x", x },
+				{ "y", y }
+			});
+		}
+
+		/// <summary>
+		/// Performs a long mouse click on the matched element.
+		/// </summary>
+		/// <param name="app">Represents the main gateway to interact with an app.</param>
+		/// <param name="element">Target Element.</param>
+		public static void LongPress(this IApp app, string element)
+		{
+			var elementToClick = app.FindElement(element);
+			app.CommandExecutor.Execute("longPress", new Dictionary<string, object>
+			{
+				{ "element", elementToClick },
+			});
+		}
+
+		/// <summary>
+		/// Performs a continuous touch gesture on the matched element.
+		/// </summary>
+		/// <param name="app">Represents the main gateway to interact with an app.</param>
+		/// <param name="element">Target Element.</param>
+		public static void TouchAndHold(this IApp app, string element)
```

Comment:
It uses **Touch** gestures and can be a direct substitute for https://learn.microsoft.com/en-us/dotnet/api/xamarin.uitest.android.androidapp.touchandhold?view=xamarin-uitest-sdk